config/initializers/service_registry.rb

```diff
@@ -0,0 +1,23 @@
+module DefenceRequestServiceRota
+  def self.services(name, service = nil)
+    @services ||= {}
+
+    if service
+      @services[name] = service
+      return true
```

Comment:
Could we return the service instead of true here?
That way the return object from both code paths is constantly the service